src/components/signup/SignupForm.react.js

```diff
@@ -37,7 +36,7 @@ const SignupForm = (props: Object) => {
   /* eslint-enable */
 
   return (
-    <Paper zDepth={1} className="signupForm">
+    <Paper zDepth={1} className="form">
```

Comment:
Add a button and action to switch from signup to login (ex. I accidentally click signup and need to get back to login form)